portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesInterests.testcase

```diff
@@ -40,47 +40,11 @@ definition {
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8283 | Test Summary: Site overview interests card shows what the top interests are"
 	@priority = 5
 	test ShowsSitesTopInterests {
-		var pageNameList = "sports,technology,science";
```

Comment:
Hey @joshitagaki, I think what we can do in this test is continue with the current version that's in the master and modify the name of these pages to be big names that can count as interest and check the name of those interests at the end of the test in the filter from Yesterday as you had modified

This test will not work on CI as we need to wait a day for these interests to appear in the Yesterday filter (Ignore the test)

Changes made in this PR that I think could continue:
- Inform the name of the card that must be checked for the selected time filter;
- Switch to Yesterday's filter;

What do you think?

Reply:
I don't think ignoring the test as a whole is a good idea just because we don't have any interests tests that would check something like the card or time filter works as intended. What I think we can do though is comment out the lines of code dealing with generating the interests. (The code I deleted here). What do you think?

Reply:
I think these two tests can be covered by a single automation as they both need interests to appear on the card (case of this automation):
- Site overview interests card shows what the top interests are
- Assert Sites Interests card highest interests

If we only want to validate that the time filter works, then we can create a separate test just for that to verify that no error occurs on the card when modifying the time filter which would be the case to cover this test case (it would not be necessary to have interests for the purpose of the test):
- Assert Sites Interests change time filter

But the name and description of this automation is a test case to verify that the interests appear (something that we cannot cover in the automation), so we cannot modify this automation to only check the time filter if that is not the objective. I think this automation should not have been used to cover all these test cases. I think we can create a new test case for the time filter and ignore this existing automation

What do you think?

Reply:
Thanks @ViniLopes20 I agree. I'll ignore this test and add a new test with just the change time filter.